Src/WebPrac/Web.config

```diff
@@ -5,7 +5,7 @@
   -->
 <configuration>
   <connectionStrings>
-    <add name="MyConnString" connectionString="Server=.\SQLEXPRESS20142;Database=EADProj;User Id=sa;Password=12345;"/>
+    <add name="MyConnString" connectionString="Server=.\SQLExpress;Database=EADProj;User Id=sa;Password=123;"/>
```

Comment:
We should not commit/push web.config file. We should change on our local machine and uncheck it while commit/pushing.

Reply:
ok sir